packages/flare/package.json

```diff
@@ -15,6 +15,7 @@
   "devDependencies": {
     "@babel/core": "^7.2.0",
     "@splunk/configuration-screen": "^0.0.1",
+    "@splunk/status-screen": "^0.0.1",
```

Comment:
I just noticed this... Our screens are different packages?

Two questions:

Should these be in devDependencies?
Should these be under an @flare/ namespace?

Reply:
I'm watching a Lerna vid right now cuz I know nothing about it.

Reply:
good point about the flare namespace, i will change it. It needs to be under the dependencies since we would also need it for a production build

Reply:
Right. I should have phrased the question as "Should these be under dependencies?" since it's currently under devDependencies.